src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3174,16 +3183,18 @@ namespace winrt::TerminalApp::implementation
             original->SetActive();
         }
 
-        _RegisterPaneEvents(resultPane);
+        _RegisterPaneEvents(terminalPane);
 
         return resultPane;
     }
 
-    void TerminalPage::_restartPaneConnection(const std::shared_ptr<Pane>& pane)
+    void TerminalPage::_restartPaneConnection(
+        const TerminalApp::TerminalPaneContent& paneContent,
+        const winrt::Windows::Foundation::IInspectable&)
```

Comment:
```suggestion
)
```
What do we need this for?

Reply:
It's a typed event that we're just plumbing through. `RestartTerminalRequested` doesn't currently actually raise the event with `args` but it could? I dunno, that's just how these are generally done.